src/cli/NativeMsgIface.cpp

```diff
@@ -359,13 +370,21 @@ request_variant NativeMsgIface::parseJsonInput(QByteArray input) {
         ret.id = id;
         for (auto&& key : mandatoryKeysDownload) {
             QJsonValueRef val = data[key];
-            if (val.isNull()) {
+            if (val.isNull() || val.isUndefined()) {
```

Comment:
So... `isNull()` is like `key: ""``, whereas `isUndefined()` is the key is not found at all?

Reply:
> The returned [QJsonValue](https://doc.qt.io/qt-6/qjsonvalue.html) is [QJsonValue::Undefined](https://doc.qt.io/qt-6/qjsonvalue.html#Type-enum) if the key does not exist.

https://doc.qt.io/qt-6/qjsonobject.html#value

Reply:
How was it then working beforehand!?